arduino-ide-extension/package.json

```diff
@@ -157,10 +158,10 @@
   ],
   "arduino": {
     "cli": {
-      "version": "0.21.0"
+      "version": "20220527"
     },
     "fwuploader": {
-      "version": "2.0.0"
+      "version": "2.2.0"
```

Comment:
Close https://github.com/arduino/arduino-fwuploader/issues/141 if the error does not happen anymore.